internal/collector/collector.go

```diff
@@ -153,7 +138,6 @@ func (f *Factory) NewPodSpec(trustedCABundle *v1.ConfigMap, forwarderSpec loggin
 
 	if f.isDaemonset {
 		podSpec.Volumes = append(podSpec.Volumes,
-			v1.Volume{Name: logContainers, VolumeSource: v1.VolumeSource{HostPath: &v1.HostPathVolumeSource{Path: logContainersValue}}},
```

Comment:
Why remove for DS?  Isn't this meant to be for deployment only?

Reply:
`/var/log/containers` path is deprecated in favor of `/var/log/pods` and has been for a long time.  We have no reason to mount it